.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.49.0
+current_version = 0.50.0
```

Comment:
we edit the version with bumpversion, and it create the tag too so leave it to me, i'll do it